src/main/resources/META-INF/resources/primefaces/core/core.env.js

```diff
@@ -32,7 +32,7 @@ if (!PrimeFaces.env) {
          */
         init : function() {
             this.browser = $.browser;
-            this.mobile = this.browser.mobile;
+            this.mobile = (this.browser.mobile) ? true : false;
```

Comment:
Always force env to have a true or false value. Right now on desktop its `undefined`.

Reply:
https://api.jquery.com/jquery.browser/
Is this something we still can rely on?